src/graphql/queries/etchPacket.js

```diff
@@ -2,7 +2,18 @@ const defaultResponseQuery = `{
   id
   eid
   name
+  status
+  isTest
+  isFree
+  containsFillData
+  payload
+  numberRemainingSigners
   detailsURL
+  webhookURL
+  createdAt
+  updatedAt
+  archivedAt
+  completedAt
```

Comment:
These are all the things that are possible. Do we want them all?

Reply:
I'd keep 

`status, isTest, numberRemainingSigners, webhookURL, *At`